### PR TITLE
Updated ReadME

### PR DESCRIPTION
**Pull request by amitkrishna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,12 +320,17 @@ To contribute, please check out our contribution guide [here](https://github.com
 - [humaaans](https://humaaans.com/)
 - [absurd](https://absurd.design/)
 
+---
```
Comment (shahednasser) [inline README.md:323]: Please remove these horizontal lines

File: README.md [hunk 2]
```diff
@@ -309,12 +320,17 @@ To contribute, please check out our contribution guide [here](https://github.com
...
 - [absurd](https://absurd.design/)
 
+--- 
+
 ### Cheat Sheets
 
 - [Markdown](https://www.markdownguide.org/cheat-sheet/)
 - [Emmet](https://docs.emmet.io/cheat-sheet/)
 
+---
```
Comment (shahednasser) [inline README.md:330]: Please remove these horizontal lines

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Please do not remove all the horizontal lines, just the ones I labeled.

**Comment by amitkrishna** [comment]
Hi @shahednasser can u review this PR?

**Comment by shahednasser** [comment]
thank you!

**Pull request merged**